src/main/java/com/auth0/exception/APIException.java

```diff
@@ -136,7 +136,8 @@ private static String obtainExceptionMessage(Map<String, Object> values) {
             Object description = values.get("description");
             if (description instanceof String) {
                 return (String) description;
-            } else {
+            } else if (description instanceof Map){
+                @SuppressWarnings("unchecked")
                 PasswordStrengthErrorParser policy = new PasswordStrengthErrorParser((Map<String, Object>) description);
                 return policy.getDescription();
             }
```

Comment:
else...? 
there should always be one default case. Maybe we could set a default message/description value here?

Reply:
I think that makes sense. I'll update with a default if the description is not a string or a map.

Reply:
Actually, I did add a test for this very case (where the description is not a string or a map):

https://github.com/auth0/auth0-java/blob/eb1288f4370fb6651eacfc559ba788eeb2afe43e/src/test/java/com/auth0/exception/APIExceptionTest.java#L63

It verifies that in that case (which really should never happen, and as of today would just cause a runtime class cast exception) the error handling falls through the rest of the condition checks and returns "Unknown exception". I think that's the correct behavior; if the description is not one of the known types (map or string), we just continue to check for other values to return (message and error), before returning "Unknown exception".